service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsController.java

```diff
@@ -35,6 +35,7 @@ public HearingBookingDetailsController(MapperService mapperService, HearingBooki
 
     @PostMapping("/about-to-start")
     public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody CallbackRequest callbackrequest) {
+
```

Comment:
you need the blank line? Less file changes 👀

Reply:
Done.